pages/architecture/session_tokens_rfc.rst

```diff
@@ -0,0 +1,152 @@
+.. _nuts-documentation-session-tokens:
+
+RFC: Session Tokens
```

Comment:
The RST document should have 1 main header.

Suggested:

H1: #####
H2: *******
H3: =====